### PR TITLE
fix(sort-messages-hil): sort order of messages in HIL

### PR DESCRIPTION
**Pull request by jaycosaur**
Sick of seeing these out of order. Quick fix.

### CODE REVIEW

File: src/hil/components/message-list/index.tsx [hunk 1]
```diff
@@ -27,10 +27,11 @@ export class MessageList extends React.Component<IMessageListProps> {
 
   public render() {
     const props = this.props;
+    const messagesOrdered = props.messages.sort((a, b) => a.timestamp - b.timestamp);
```
[the remark this answers is not in the document]
Comment (jaycosaur) [reply src/hil/components/message-list/index.tsx]: Should probably sort them on the server first too.

[the remark this answers is not in the document]
Comment (zaripych) [reply src/hil/components/message-list/index.tsx]: We cannot sort them on server for an ongoing conversation.

[the remark this answers is not in the document]
Comment (jaycosaur) [reply src/hil/components/message-list/index.tsx]: They don't go out of order for an ongoing convo only if something goes wrong (so will still need it in the reducer), it's only out of order at the moment when using the loadConversation.

[the remark this answers is not in the document]
Comment (zaripych) [reply src/hil/components/message-list/index.tsx]: Then maybe fix it on server side rather than here.

**Pull request closed without merging**